plugins/plugin-docker/che-plugin-docker-machine/src/main/java/org/eclipse/che/plugin/docker/machine/MachineProviderImpl.java

```diff
@@ -132,6 +132,7 @@
     private final long                                          cpuPeriod;
     private final long                                          cpuQuota;
     private final WindowsPathEscaper                            windowsPathEscaper;
+    private final String                                        volumesFromContainerName;
```

Comment:
It id better to call it volumesFrom because it is convenient term in docker ecosystem